content/en/docs/mechanical-assembly/belt-tensioners/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Belt Tensioners"
+linkTitle: "Belt Tensioners"
+weight: 9
+resources:
+- src: "**.png"
+- src: "**.jpg"
+description: >
+  Assembling the belt tensioners used throughout the machine
+---
+
+Alright, time to get into the build! First, we're going to build the belt tensioners that are used for the X and Y axes. You need to build **three** of these.
+
+| Qty              | Part                          |
+|-----------------:|-------------------------------|
+| 1 each, 3 total  | M3 Hex Nut                    |
+| 1 each, 3 total  | M3x16 machine screw           |
+| 1 each, 3 total  | M3 Cap Nut                    |
+| 1 each, 3 total  | M5x25 machine screw           |
+| 1 each, 3 total  | M5 Nyloc Hex Nut              |
+| 1 each, 3 total  | M5x40 machine screw           |
+| 1 each, 3 total  | M5 Hex Nut                    |
+| 1 each, 3 total  | GT2 Pulley Idler              |
+| 1 each, 3 total  | FDM-0037 (Belt Tension Arm)   |
+| 1                | FDM-0001 (Front Left Leg)     |
+| 1                | FDM-0002 (Front Right Leg)    |
+| 1                | FDM-0013 (X Idler Mount)      |
+
+## Assembling the tensioner arms
+
+{{< container-image path="images/1-belt-tension-arm-parts.jpg" alt="Belt Tension Arm Parts" >}}
+
+1. Insert an M3 Hex Nut into its slot on the back of a {{< tooltip "Belt Tension Arm" "FDM-0037" >}}.
+  {{< container-image path="images/2-m3-nut-inserted.jpg" alt="Inserted M3 Hex Nut" >}}
+
+2. Insert an M3x16mm Machine Screw through the front hole in the {{< tooltip "Belt Tension Arm" "FDM-0037" >}} and then tighten.
```

Comment:
**I just realized this is a problem. The tensioner arm that goes into the left foot vs. the right food has one of the screws reversed (Steps 2 and 3). This is a problem with the screenshots I have from the current live master build, too.**

So, unfortunately, the 3 tensioner arms are not exactly identical :( 

If you don't reverse the direction of that screw, the screw that goes through the idler will be flipped around, with the screw head facing down instead of up. Honestly, I don't think that's awful. I don't think that affects function of the machine apart from making that screw a tiny bit more difficult to loosen if necessary...

Reply:
@sphawes can you confirm if this matters at all? I was planning on just having people install the belt tensioner arm upside down and call it a day, but if there's a problem with that, we should rework these steps.